steps/set-terraform-varfile-env-var.yaml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  terraformVarFile: ''
+  environment: ''
+
+steps:
+  - pwsh: |
```

Comment:
why powershell? every other script we use is bash

Reply:
...would be updated 👌🏾

Reply:
yaml step that has powershell is removed, in favour of having logic in yaml template